### ISSUE TITLE
Aim telemetry hangs in air-gapped environment

### ISSUE THREAD

**Issue by wlhjason** [aimhubio/aim#2539]
## 🐛 Bug

Within an environment with no internet access, the Aim SDK can hang on exit while Segment attempts to flush telemetry data to its servers.

### To reproduce

Create a `run_aim.py` script:

```python
import aim
import logging

logging.basicConfig()
run = aim.Run()
```

and run `python run_aim.py`.

This hangs on exit and produces log messages like this:

```
INFO:backoff:Backing off send_request(...) for 0.8s (requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.segment.io', port=443): Max retries exceeded with url: /v1/batch (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fe251a0f190>, 'Connection to api.segment.io timed out. (connect timeout=15)')))
INFO:backoff:Backing off send_request(...) for 1.8s (requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.segment.io', port=443): Max retries exceeded with url: /v1/batch (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fe251a2efa0>, 'Connection to api.segment.io timed out. (connect timeout=15)')))
INFO:backoff:Backing off send_request(...) for 1.4s (requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.segment.io', port=443): Max retries exceeded with url: /v1/batch (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fe2519ba130>, 'Connection to api.segment.io timed out. (connect timeout=15)')))
```

### Environment

- Aim Version: **3.6.0** (following https://github.com/aimhubio/aim/pull/2490/)
- Python version: 3.8.13
- pip version: 22.2.2
- OS: Ubuntu 20.04.4

### Additional context

Manually disabling Aim telemetry beforehand via `$ aim telemetry off` (within the same Python environment) resolves the issue, but it would be great if this wasn't necessary!

**Comment by SGevorg** [comment]
Thanks for sharing this @wlhjason. Looking into it. 
Adding @alberttorosyan and @mihran113 to this thread as well.

**Comment by wlhjason** [comment]
Thanks for the quick response!

As a related issue, I noticed that `analytics.track_install_event()` (which is called at import time) calls out to Segment if it detects that Aim is running in a new Python environment. Thus:

* `$ aim telemetry off` can hang indefinitely when run within a new Python environment.

* There is no way to completely disable telemetry at the moment.

**Comment by wlhjason** [comment]
@SGevorg I’m able to use Aim 3.15, which does work without internet access

**Comment by wlhjason** [comment]
Can confirm that this has been fixed as of Aim 3.16.1 - thanks @alberttorosyan!
